config/certmanager/llmisvc/certificate.yaml

```diff
@@ -20,5 +20,5 @@ spec:
     - $(webhookServiceName).$(kserveNamespace).svc
   issuerRef:
     kind: Issuer
-    name: selfsigned-issuer
+    name: llmisvc-selfsigned-issuer
```

Comment:
I'm wondering if this is intentional to require a separate issuer for llmisvc and what's the reasoning or if we just use the same issuer for all internal certs

Reply:
You're right. updated.